src/core/icons/folderIcons.ts

```diff
@@ -584,7 +613,14 @@ export const folderIcons: FolderTheme[] = [
       { name: 'folder-ansible', folderNames: ['ansible'] },
       {
         name: 'folder-server',
-        folderNames: ['server', 'servers', 'backend', 'backends'],
+        folderNames: [
+          'server',
+          'servers',
+          'backend',
+          'backends',
+          'internal',
```

Comment:
I'm not entirely sure about the choice to use the folder name "internal" here. Could we clarify the intention behind it? The term "internal" might suggest something private or restricted, which could be a bit misleading. Also, I'm not convinced the server folder icon clearly communicates this meaning. Perhaps we could consider removing it for now or exploring an alternative?